lib/Importer/ONNX.cpp

```diff
@@ -98,6 +98,16 @@ bool ONNXModelLoader::loadProto(ONNX_NAMESPACE::GraphProto &net,
   std::ifstream ff(filename, std::ios::in | std::ios::binary);
   GLOW_ASSERT(ff && "Can't find the model or network files.");
 
+  if (filename.find(".onnxtxt") != std::string::npos) {
+    std::string str((std::istreambuf_iterator<char>(ff)),
+                    std::istreambuf_iterator<char>());
+    ONNX_NAMESPACE::ModelProto MP;
```

Comment:
Is there any way if this code could open file as a stream, and call `loadProto(net, fileStream)`? Am worried about code duplication between this and a function above. `net = MP.graph();` and `setVersion(MP);` are duplicated. I know it's not a big deal for now, but what if want to add more initialization stuff in future?

Reply:
Haven't found a proper way to reuse loadProto(net, filestream). Will mark this one as TODO.